webapp/package.json

```diff
@@ -13,20 +13,21 @@
     "test-ci": "jest --forceExit --detectOpenHandles --maxWorkers=2"
   },
   "dependencies": {
+    "core-js": "^3.19.1",
```

Comment:
We shouldn't have core-js as a direct dependency. Could we `npm uninstall it`?

I'm guessing adding was partly an attempt to get babel to upgrade the version it needs and pulls in. It looks like babel's latest version is 7.16.0. Could we try upgrading `@babel/cli` and all the `@babel/plugin-*` packages to 7.16.0 (the babel plugin packages usually move lockstep in version with babel)?

Reply:
I stand corrected. This _is_ the right approach: https://babeljs.io/docs/en/babel-polyfill

Reply:
Approach followed in commit b599688